resources/Components/Countdown/index.js

```diff
@@ -1,7 +1,6 @@
-import React from 'react'
 import useCountdown from '../../Hooks/useCountdown'
 
-export default ({ end, title, titleClass, innerClass }) => {
+export const Countdown = ({ end, title, titleClass, innerClass }) => {
```

Comment:
let's not change the API of exported modules where we can avoid it, if we want to retain the naming property of this component in the React dev tools we can use:
```
const Countdown = () => { ... }
export default Countdown
```